### ISSUE TITLE
Visualize and Export buttons throws errors

### ISSUE THREAD

**Issue by stigok** [Synthoid/ExportSheetData#37]
Currently logs an error in the console
```
Uncaught Error: Authorization is required to perform that action.
```

Unfortunately I'm not sure how to add more information to this bug report, but the add-on does not currently work at all.

**Comment by Synthoid** [comment]
Hmm. Do you own the document that you are trying to export?

**Comment by stigok** [comment]
- Removing the addon from the current document and re-adding does not help.
- Removing the addon by "uninstalling" it from the Drive account then re-adding does not help.

**Comment by stigok** [comment]
Yes, the document is *owned by me*. It does not make a difference whether or not I have selected "Replace existing file(s)"

**Comment by Synthoid** [comment]
Also, are you using a G Suite account?

**Comment by stigok** [comment]
No, I am using a standard Google account.

**Comment by Synthoid** [comment]
Interesting. Did you agree to the permissions prompt when installing ESD? It should ask you for permission to control your Drive files.

**Comment by stigok** [comment]
I did remove the addon from the *Manage add-ons* pane, then installing it again. It asks me for permissions which I have to accept before continuing. Everything seems to go well until attempting to use the add-on.

**Comment by stigok** [comment]
It was resolved after re-attempting in a new browser session. I was logged in to two different Google accounts, of which the last one I logged in with was the owner of the Drive instance I was using. This seems to be an issue with Google Drive itself.

The solution seems to be to log out of active Google accounts then log in again only with a single desired user account and see if the problem disappears.
